readme.md

```diff
@@ -11,6 +11,12 @@ This is for advanced users. You probably want to use XO directly.
 $ npm install --save-dev eslint-config-xo
 ```
 
+For the `esnext` version you'll also need Babel's eslint parser and plugin:
+
+```
+$ npm install --save-dev babel-eslint eslint-plugin-babel
+```
```

Comment:
Couldn't agree more!
In the case of XO, XO itself will depend on `babel-eslint` and `eslint-plugin-babel` to solve this (in my upcoming PR). What do you think about that?